.bit/responses/3.2-Week 3 Step 2.md

```diff
@@ -1,109 +1,75 @@
 ---
-files: js/databaseContext.js
+files: n/a
 stepType: checks
-scripts: test.databaseContext.js
+scripts: test.secret1.js
 week: 3
 step: 2
 name: Week 3 Step 2
 ---
 
 ## Week 3 Step 2
 
-### Setting Up Configuration
+### Calling the Giphy API
 
-Now that we've created our Cosmos accout and database, we now need to set up our global variable configuration in code. Navigate to your source code in **VSCode**. 
+Open up the **HTTP trigger function** that you made last week. When it is run, the function should call the **Twilio API**. In this step, we're going to call the **Giphy API** in the trigger function!
 
-1. Create a file named `config.js`.
-2. Collect the following pieces of information:
-   1. Your Cosmos DB Endpoint (this should look something like `https://yourcosmosaccountname.documents.azure.com:443/`)
-   2. Your Subscription Key (Navigate to your cosmos db account page, go to **Overview** and find **Subscription ID**)
-   3. Your Database ID (look at your previous comment!)
-   4. Your Container ID (look at your previous comment!)
+*Note: do **NOT** use the first trigger function you coded in Week 1 – make sure you are working in the **Twilio trigger function!** Both should be in the same resource group, but they are **different** trigger functions!*
 
-<p align="center"><img src="https://user-images.githubusercontent.com/49426183/113173610-906e8f80-9217-11eb-940e-aa669a45131b.png"></p>
+<details>
+<summary>:question: Where is the documentation again?</summary>
+  </br>
+The documentation is here: https://developers.giphy.com/docs/api/endpoint#translate. This link has everything you need to call the **Giphy API.**

```

Comment:
Make sure to add a space after </br> or else the markdown inside the dro pdown won't work